civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -5015,6 +5015,12 @@ function civihr_employee_portal_user_login(&$edit, $account) {
  * @return bool
  */
 function _civihr_employee_portal_should_do_onboarding($account) {
+  $userWantsToTest = isset($_GET['testing']) && $_GET['testing']==1;
```

Comment:
Not sure about the formatting here, think there should be spaces around `==`

Reply:
You are right, I will correct it.